.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
you can add https://npmjs.com/eccheck to CI to verify this, so it doesn't deviate

Reply:
Sorry, link above triggers 404 for me, is it some private package?

Reply:
oops sorry, https://www.npmjs.com/package/eclint :-)

Reply:
That's quite a nice package, have not been aware of this, thanks!
I will play around and see if I can add it in the same PR